pkg/storaged/block/format-dialog.jsx

```diff
@@ -406,6 +412,8 @@
                 } else {
                     dlg.update_actions({ Variants: [action_variant], Title: action_title });
                 }
+                if (vals.type == "efi" && !vals.mount_point)
+                    dlg.set_values({ mount_point: "/boot/efi" });
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19826-20240109-133522-cb1438ac-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)